packages/apollo-language-server/src/providers/schema/index.ts

```diff
@@ -25,19 +25,17 @@ export function schemaProviderFromConfig(
   config: ApolloConfig,
   clientIdentity?: ClientIdentity // engine provider needs this
 ): GraphQLSchemaProvider {
-  if (isServiceConfig(config)) {
-    if (config.service.localSchemaFile) {
-      const isListOfSchemaFiles = Array.isArray(config.service.localSchemaFile);
-      return new FileSchemaProvider(
-        isListOfSchemaFiles
-          ? { paths: config.service.localSchemaFile as string[] }
-          : { path: config.service.localSchemaFile as string }
-      );
-    }
+  if (config.service && config.service.endpoint) {

```

Comment:
Why was the order of these flipped again? I remember there was a distinct reason, but I don't remember exactly what it was